integreat_cms/cms/templates/pois/poi_form.html

```diff
@@ -213,16 +213,34 @@ <h1 class="heading">
-                                    data-action="{% url 'delete_poi' poi_id=poi_form.instance.id region_slug=region.slug language_slug=language.slug %}">
-                                    <i data-feather="trash-2" class="mr-2"></i> {% trans 'Delete this location' %}
-                                </button>
-                            </div>
+                                <div>
+                                    <label>{% trans 'Delete location' %}</label>
+                                    {% if poi_form.instance.events.exists == False %}
+                                        <button title="{% trans 'Delete location' %}" class="btn confirmation-button w-full btn-red"
+                                            data-confirmation-title="{{ delete_dialog_title }}"
+                                            data-confirmation-text="{{ delete_dialog_text }}"
+                                            data-confirmation-subject="{{ poi_form.instance|poi_translation_title:language }}"
+                                            data-action="{% url 'delete_poi' poi_id=poi_form.instance.id region_slug=region.slug language_slug=language.slug %}">
+                                            <i data-feather="trash-2" class="mr-2"></i> {% trans 'Delete this location' %}
+                                        </button>
+                                     {% else %}
+                                        <div class="bg-orange-100 border-l-4 border-orange-500 text-orange-500 px-4 py-3 mb-5" role="alert">
+                                            <p>
+                                                {% trans 'You cannot delete a location which is referenced by an event.' %}<br>
+                                                {% blocktrans count counter=poi_form.instance.events.count trimmed %}
+                                                    To delete this location, you have to delete this event first:
+                                                {% plural %}
+                                                    To delete this location, you have to delete these events first:
+                                                {% endblocktrans %}
+                                            </p>
+                                        </div>
+                                        {% for event in poi_form.instance.events.all %}
+                                            <a href="{% url 'edit_event' event_id=event.id region_slug=region.slug language_slug=language.slug %}" class="block pt-2 hover:underline">
+                                                <i data-feather="edit" class="mr-2"></i> {{ event.best_translation.title }}
+                                            </a>
+                                        {% endfor %}
+                                    {% endif %}  
+                                </div>
```

Comment:
```suggestion
                            <div>
                                <label>{% trans 'Delete location' %}</label>
                                {% if poi_form.instance.events.exists %}
                                    <div class="bg-orange-100 border-l-4 border-orange-500 text-orange-500 px-4 py-3 mb-5" role="alert">
                                        <p>
                                            {% trans 'You cannot delete a location which is referenced by an event.' %}<br>
                                            {% blocktrans count counter=poi_form.instance.events.count trimmed %}
                                                To delete this location, you have to delete this event first:
                                            {% plural %}
                                                To delete this location, you have to delete these events first:
                                            {% endblocktrans %}
                                        </p>
                                    </div>
                                    {% for event in poi_form.instance.events.all %}
                                        <a href="{% url 'edit_event' event_id=event.id region_slug=region.slug language_slug=language.slug %}" class="block pt-2 hover:underline">
                                            <i data-feather="edit" class="mr-2"></i> {{ event.best_translation.title }}
                                        </a>
                                    {% endfor %}
                                {% else %}
                                    <button title="{% trans 'Delete location' %}" class="btn confirmation-button w-full btn-red"
                                            data-confirmation-title="{{ delete_dialog_title }}"
                                            data-confirmation-text="{{ delete_dialog_text }}"
                                            data-confirmation-subject="{{ poi_form.instance|poi_translation_title:language }}"
                                            data-action="{% url 'delete_poi' poi_id=poi_form.instance.id region_slug=region.slug language_slug=language.slug %}">
                                        <i data-feather="trash-2" class="mr-2"></i> {% trans 'Delete this location' %}
                                    </button>
                                {% endif %}
                            </div>
```
If you have an if condition, I'd always prefer simpler conditions... So I'd list the `True` case first.